notebook/static/notebook/js/notebook.js

```diff
@@ -2710,10 +2728,10 @@ define([
                           that._changed_on_disk_dialog = dialog.modal({
                             notebook: that,
                             keyboard_manager: that.keyboard_manager,
-                            title: "Notebook changed",
-                            body: "The notebook file has changed on disk since the last time we opened or saved it. "+
-                                  "Do you want to overwrite the file on disk with the version open here, or load "+
-                                  "the version on disk (reload the page) ?",
+                            title: i18n._("Notebook changed"),
+                            body: i18n._("The notebook file has changed on disk since the last time we opened or saved it.") + " " +
+                                  i18n._("Do you want to overwrite the file on disk with the version open here, or load "+
+                                  "the version on disk (reload the page) ?"),
```

Comment:
Is there a reason to separate these two sentences rather than translating them as one unit? I assume that one chunk of text can be translated more naturally.